packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -99,8 +99,7 @@ const ProductDetail = () => {
                     'prices',
                     'variations',
                     'set_products',
-                    'bundled_products',
-                    'page_meta_tags'
+                    'bundled_products'
```

Comment:
Are we reverting the page meta tag work? If so, then we'll need to update the retail-react-app's changelog accordingly, as it currently says:
> - PDP / PLP: Add page meta data tags that have been defined in BM [#2232](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2232)

Reply:
Nope.. you caught a bad merge. I'll update that.